_posts/2015-09-09-the-plain.markdown

```diff
@@ -0,0 +1,14 @@
+---
+layout: post
+title:  "The Plain"
+date:   2015-09-09 10:23:00
```

Comment:
Hello! Please fix formatting in front matter, by removing extra quotes and whitespaces:

```
title: The Plain
date: 2015-09-09 10:23:00
```